doc/source/rllib/rllib-torch2x.rst

```diff
@@ -0,0 +1,134 @@
+.. include:: /_includes/rllib/announcement.rst
+
+.. include:: /_includes/rllib/we_are_hiring.rst
+
+
+
+Using RLlib with torch 2.x compile
+==================================
+
+Torch 2.x comes with the new ``torch.compile()`` `API <https://pytorch.org/docs/stable/generated/torch.compile.html#torch.compile>`_, which leverages `torch dynamo <https://pytorch.org/docs/stable/dynamo/index.html#torchdynamo-overview>`_ under the hood to JIT-compile wrapped code. We integrate ``torch.compile()`` with RLlib in the context of `RLModules <rllib-rlmodule.html>`_ and Learners. 
+
+We have integrated this feature with RLModules. You can set the backend and mode via ``framework()`` API on an :py:class:`~ray.rllib.algorithms.algorithm_config.AlgorithmConfig` object. Alternatively, you can compile the :py:class:`~ray.rllib.core.rl_module.rl_module.RLModule` directly during stand-alone usage such as inference.
+
+Benchmarks
+==========
+
+We conducted a comprehensive benchmark with this feature. 
```

Comment:
Especially since above we say that "You can set the backend and mode via ``framework()`` API [...]" which would require user to understand on their own that they should not set the backend for the learner but _only_ for the worker, that they can choose distinct backends here and so on. To make this hyper-accessible, we should point to the `TorchRLModule.compile()` API for inference as well.